### ISSUE TITLE
Bitmap memory request failed

### ISSUE THREAD

**Issue by fengxue-jrql** [ptitSeb/box86#686]
Bitmap memory request failed

**Comment by fengxue-jrql** [comment]
![位图内存不足](https://user-images.githubusercontent.com/57759830/194746398-419eeace-d69b-44f0-9eae-ea61a815151d.jpg)
When I run Windows games through box86 and wine, some games will display Bitmap memory request failed, which will cause the game to crash, and even some games cannot run because of this problem.

**Comment by ptitSeb** [comment]
Can you give me a an example of game I can download somewhere
Also, what Wine version are you using?

**Comment by fengxue-jrql** [comment]
"カノジョ*ステップ" and "Making*Lovers" (Japanese: メイキング*ラバーズ) Two Chinese versions of Japanese beauty girl games will collapse due to Bitmap memory request failed.
wine is winedevel-7.10

**Comment by fengxue-jrql** [comment]
Please consider allowing box86 and box64 to request larger bitmap memory

**Comment by fengxue-jrql** [comment]
Bitmap memory request failed is on the game《クロスコンチェルト》，when i run this game with long time.

**Comment by ptitSeb** [comment]
It may be a bug of the graphic driver also. Box86/Box64 doesn't do anything special with Bitmap memory, it just translate Wine funciton calls.
On what hardware are you running this?

**Comment by fengxue-jrql** [comment]
I run box86 and box64 with wine7.10 on arm64 kali by linux deploy.It's opengl provide by llvmpipe,vulkan is provide by lavapipe.

**Comment by ptitSeb** [comment]
Are you running on Termux with proot?

**Comment by fengxue-jrql** [comment]
no, i was run box86 box64 and wine7.10 on linux deploy chroot kali(arm64)

**Comment by ptitSeb** [comment]
Ok,
well, I'm not sure if it's an actual issue of box86 or something else (like an issue with Wine).

It would be interresting to check if the same game with the same Wine version (and the same graphic driver, if it's software llvmpipe) works fine on an real x86 linux.

**Comment by ThieuMinh26** [comment]
> I run box86 and box64 with wine7.10 on arm64 kali by linux deploy.It's opengl provide by llvmpipe,vulkan is provide by lavapipe.

is there any different between with llvmpipe and lavapipe

**Comment by ThieuMinh26** [comment]
> ![位图内存不足](https://user-images.githubusercontent.com/57759830/194746398-419eeace-d69b-44f0-9eae-ea61a815151d.jpg) When I run Windows games through box86 and wine, some games will display Bitmap memory request failed, which will cause the game to crash, and even some games cannot run because of this problem.

and thats fast , how did you get that much fps
